### ISSUE TITLE
Account details password fields feature broken plain-text togglers

### ISSUE THREAD

**Issue by slavicd** [woocommerce/storefront#1290]
## Describe the bug
Clicking the plain-text toggler on the password fields will display the input as plain text, however further clicks will do nothing.

Isolating the problem (mark completed items with an [x]):
- [x] I have deactivated other plugins and themes and confirmed this bug occurs when only WooCommerce + Storefront theme are active.
- [x] I can reproduce this bug consistently using the steps below.

## To Reproduce
Steps to reproduce the behavior: toggle twice any input from plain text to password mode

## Expected behavior
Upon second toggle the password should turn back to asterisks.

**Comment by haszari** [comment]
Hi @slavicd – can you provide more details of your setup?

I tried this in Firefox, Chrome, and Safari (on macOS) and I couldn't reproduce the issue. Here's a gif:

![show-password](https://user-images.githubusercontent.com/4167300/77382685-d82d3080-6de5-11ea-812b-73a0e13a1d96.gif)

**Comment by haszari** [comment]
Aha! While I was testing #1289 I saw this issue, I see it's in my account when logged in (I missed that detail in the issue title).

Steps to reproduce:

- Visit `My account > Account details`  page `/my-account/edit-account/`.
- Type in the `New password` field.
- Click the password visibility toggle icon to show the password.
- Click again to re-hide the password.

Expected: the password to hide, can easily toggle show/hide.

Actual: the toggle only works on the first click.

@slavicd – I've reproduced this in another theme (Twenty Nineteen), so it looks to me like this is an issue in WooCommerce core.

I've logged in Woo repo here https://github.com/woocommerce/woocommerce/issues/26001, closing this issue.
